openprescribing/measure_definitions/methotrexate.json

```diff
@@ -15,7 +15,7 @@
     "To avoid error with low-dose methotrexate, it is recommended that: ",
     "the patient is carefully advised of the dose and frequency and the reason for taking methotrexate ",
     "and any other prescribed medicine (e.g.folic acid); only one strength of methotrexate tablet ",
-    "(usually 2.5 mg) is prescribed and dispensed."
+    "(usually 2.5 mg) is prescribed and dispensed. You can read more in our paper on <a href='https://doi.org/10.3399/bjgp20X710993'> trends and variation in unsafe prescribing of methotrexate </a>."
```

Comment:
There's a stray space before the closing `</a>` tag, which will show up as space before the full stop.